### PR TITLE
Software-Manual FR update for 3.8.1

### PR DESCRIPTION
**Pull request by jujudusud**
# Does this need translation?

- [ ] Yes 
- [x] No 

# Changes

Updated Software_Manual files of FR language for 3.8.1 release.

### CODE REVIEW

File: translator-files/po/fr/Software-Manual.po [hunk 1]
```diff
@@ -241,9 +247,16 @@ msgstr "## Effet réverbération"
...
+#| "Reverb can be applied to one local mono audio channel or to both channels "
+#| "in stereo mode.  The mono channel selection and the reverberation level "
+#| "can be modified. For example, if a microphone signal is fed in to the "
+#| "right audio channel of the sound card and a reverb effect needs to be "
+#| "applied, set the channel selector to the right and move the fader upwards "
+#| "until the desired reverb level is reached."
 msgid ""
-"Reverb can be applied to one local mono audio channel or to both channels in "
-"stereo mode.  The mono channel selection and the reverberation level can be "
+"Adds reverb to your local mono audio channel, or to both channels in stereo "
```
Comment (ann0see) [inline translator-files/po/fr/Software-Manual.po:258]: Did this get produced automatically?

Comment (ignotus666) [reply translator-files/po/fr/Software-Manual.po:258]: I'm assuming @jujudusud ran po4a-update-templates, which transferred the recent changes to Software-Manual.md to the .po files and updated them. The lines starting with #| are "previous untranslated strings" and will not show up in the final document.

**Review by ignotus666 — APPROVED** [review]
Looks ok to me. I see @jujudusud ran po4a-update-templates.sh and then selectively pushed only the files related to French (not saying that's bad - in fact it avoids a cascade of changes). OTOH I think it's preferable to wait until the translation period starts "officially" to avoid potentially wasting effort translating docs that might change again, but that's up to each person I suppose.

I do however think it might be time to consider automating the scripts with gh actions so that they get run "centrally" and no one has to worry about remembering to run them or not locally. Should I open an issue perhaps? There are a number of things to consider first, I think.

**Comment by ann0see** [comment]
Ok. Feel free to open an issue. I assume these scripts can be run here: https://github.com/jamulussoftware/jamuluswebsite/blob/daf42e7d22f3da57e3bdb15cc028f8fc736c8bbb/.github/workflows/jekyll.yml#L20

and here: 

https://github.com/jamulussoftware/jamuluswebsite/blob/daf42e7d22f3da57e3bdb15cc028f8fc736c8bbb/.github/workflows/main.yml#L29

Are you ok with a merge of this PR?

**Comment by ignotus666** [comment]
> Are you ok with a merge of this PR?

You mean this one? Yes, I think it's good to go.

> I assume these scripts can be run here:
> and here:

I made the scripts to run only when they actually need to, i.e. templates only get updated if docs in wiki/en/ are edited and targets only get created if files in po/** have been edited. These and other things are what I wanted to comment on - I'll open an issue.

**Pull request merged**